_Week8/Group4/results.do

```diff
@@ -0,0 +1,34 @@
+clear all
+
+global username "/Users/geenapanzitta/Documents/GitHub"
+
+cd "${username}/econ490-fall22/_Week8/Group4"
+
+run "simulate.do"
+
+set seed 5731341 //generated using random.org
+
+foreach j in 50 100 200 300 400 500 {
+	forv i = 1/100 {
+		clear
+		qui income_sim `j'
+		mat results = nullmat(results) \ [`i', `j', `r(b_age)', `r(b_educ_yrs)', `r(b_dist_from_city)', `r(b_family_size)', `r(b_experience)']
+	}
+}
+
+mat colnames results = i n b_age b_educ_yrs b_dist_from_city b_family_size b_experience
```

Comment:
It may be more intuitive to name the columns without the leading "b_", as this would mimic how coefficients are presented/labelled in regression tables.
```suggestion
mat colnames results = i n age educ_yrs dist_from_city family_size experience
```